bazel/external/fmtlib.BUILD

```diff
@@ -1,11 +1,22 @@
+load("@rules_cc//cc:cc_library.bzl", "cc_library")
+
 licenses(["notice"])  # Apache 2
 
 cc_library(
-    name = "fmtlib",
+    name = "fmt",
+    srcs = [
+        #"src/fmt.cc", # No C++ module support, yet in Bazel (https://github.com/bazelbuild/bazel/pull/19940)
+        "src/format.cc",
+        "src/os.cc",
+    ],
     hdrs = glob([
         "include/fmt/*.h",
     ]),
-    defines = ["FMT_HEADER_ONLY"],
+    copts = select({
+        "@rules_cc//cc/compiler:msvc-cl": ["-utf-8"],
```

Comment:
as we no longer support (or test) windows build im wondering about this ... is it tested? do we need it?